aws-cpp-sdk-core/source/utils/logging/FormattedLogSystem.cpp

```diff
@@ -96,9 +96,9 @@ void FormattedLogSystem::Log(LogLevel logLevel, const char* tag, const char* for
         vsnprintf(outputBuff.GetUnderlyingData(), requiredLength, formatStr, args);
     #endif // WIN32
 
-    ss << outputBuff.GetUnderlyingData() << std::endl;  
-  
-    ProcessFormattedStatement(std::move(ss.str()));
```

Comment:
This is redundant, but certainly not pessimistic. It is a typecast from an r-value to an r-value.